sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2606,11 +2599,11 @@
         },
         "endTimeUtc": {
           "format": "date-time",
-          "description": "System generated.  Ignored at creation.\r\nRepresents the time the job stopped processing.",
+          "description": "System generated.  Ignored at creation. Represents the time the job stopped processing.",
           "type": "string"
         },
         "type": {
-          "description": "Required.\r\nThe type of job to execute.",
+          "description": "Required. The type of job to execute.",
```

Comment:
Get rid of the "Required" statement. That will live as a swagger attribute, so there's no need for it in the description

Reply:
Was planning on doing this on a separate PR that adds the 'required' attribute. But given this is in the description, it makes sense to remove it in this PR. I'll remove all instances.

Reply:
If you think it is a good reminder and want to remove it in the next PR, totally fine.